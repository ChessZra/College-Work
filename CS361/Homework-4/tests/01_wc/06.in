wc
wc wc wc
hello test

wc
