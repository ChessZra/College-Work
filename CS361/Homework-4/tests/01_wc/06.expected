cs361sh> Num Bytes: 24
Num Lines: 4
Num Words: 6
cs361sh> 
